content/projects.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Projects"
+description: ""
+summary: ""
+date: 2025-04-08T02:49:45+02:00
+lastmod: 2025-04-08T02:49:45+02:00
+draft: false
+type: "legal"
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+
+## Explore Community Projects
+
+Below you can find links to projects maintained by the OpenFusion team as well as other third-party developers. This ranges from modding and development tools, to sites that visualize in-game information for easier viewing.
+
+> While these projects are generally considered safe, we recommend exercising caution and using your best judgment when downloading or using third-party tools.
+
+### Websites
+
+- [Original / Future Public Server Map Visualizer](https://dexlabs.systems/)
+- [Academy Public Server Map Visualizer](https://academy.dexlabs.systems/)
+- [Racing Leaderboards](https://dex-labs-leaderboards.vercel.app/)
+
+### Repositories
+
+- [OpenFusion API](https://github.com/OpenFusionProject/ofapi)
+- [OpenFusionMap](https://github.com/OpenFusionProject/OpenFusionMap-rs)
+- [DexLabs Leaderboards](https://github.com/hichemfantar/dex-labs-leaderboards)
+- [Computress Discord Bot](https://github.com/OpenFusionProject/computress-rs)
+- [Scripts](https://github.com/OpenFusionProject/scripts)
+- [FFSpy](https://github.com/OpenFusionProject/FFSpy)
+- [UnityPackFF](https://github.com/dongresource/UnityPackFF)
+- [FusionFall to Unity Converter](https://github.com/Eperty123/FF2Unity)
+- ~[Disunityff (DEPRECATED)](https://github.com/OpenFusionProject/disunityff)~
+- ~[OpenFusion Server Configurator (DEPRECATED)](https://github.com/Eperty123/OpenFusion-Server-Configurator)~
+- ~[OpenFusion Race Rankings Python Script (DEPRECATED)](https://gist.github.com/CakeLancelot/6e288aecfaf9d640f5b480b48cf82a95)~
+- ~[OpenFusionMap (DEPRECATED)](https://github.com/OpenFusionProject/OpenFusionMap)~
+- ~[OpenFusion Launcher (DEPRECATED)](https://github.com/Eperty123/OpenFusion-Launcher)~
```

Comment:
Like I said in the prior review I feel this bullet point is only really going to cause confusion, please remove the other OpenFusion Launcher repo.